onnxruntime/core/providers/webnn/builders/impl/activation_op_builder.cc

```diff
@@ -31,33 +31,39 @@ Status ActivationOpBuilder::AddToModelBuilderImpl(ModelBuilder& model_builder,
   const auto& op_type(node.OpType());
   emscripten::val input = model_builder.GetOperand(node.InputDefs()[0]->Name());
   emscripten::val output = emscripten::val::object();
-  if (op_type == "Relu") {
-    if (Contains(model_builder.GetFusedActivations(), node.InputDefs()[0]->Name())) {
-      LOGS_DEFAULT(VERBOSE) << "Relu Node [" << node.Name() << "] fused";
-      output = input;
-    } else {
-      output = model_builder.GetBuilder().call<emscripten::val>("relu", input);
-    }
-  } else if (op_type == "LeakyRelu") {
-    if (Contains(model_builder.GetFusedActivations(), node.InputDefs()[0]->Name())) {
-      LOGS_DEFAULT(VERBOSE) << "LeakyRelu Node [" << node.Name() << "] fused";
-      output = input;
-    } else {
-      NodeAttrHelper helper(node);
-      emscripten::val options = emscripten::val::object();
+
+  if (Contains(model_builder.GetFusedActivations(), node.InputDefs()[0]->Name())) {
+    LOGS_DEFAULT(VERBOSE) << op_type << " Node [" << node.Name() << "] fused";
+    output = input;
+  } else {
+    NodeAttrHelper helper(node);
+    emscripten::val options = emscripten::val::object();
+    if (op_type == "Elu") {
+      options.set("alpha", helper.Get("alpha", (float)1.0));
```

Comment:
Why not just `1.0f`? The C-style cast looks weird.

Reply:
Ah, just quickly copy/paste, fixed.